common-template/src/test/java/ms_teams/TestOcmTemplate.java

```diff
@@ -4,51 +4,74 @@
 import jakarta.inject.Inject;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import java.util.Map;
 
 import static com.redhat.cloud.notifications.qute.templates.IntegrationType.MS_TEAMS;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_APP_NAME;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CAPACITY_MANAGEMENT;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_ACCESS;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_ADD_ON;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_CONFIGURATION;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_LIFECYCLE;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_NETWORKING;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_OWNERSHIP;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_SCALING;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_SECURITY;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_SUBSCRIPTION;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CLUSTER_UPDATE;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_CUSTOMER_SUPPORT;
+import static com.redhat.cloud.notifications.qute.templates.mapping.OpenShift.CLUSTER_MANAGER_GENERAL_NOTIFICATION;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @QuarkusTest
 class TestOcmTemplate {
 
     private static final JsonObject MESSAGE = OcmTestHelpers.createOcmMessage("Atlantic", "OSDTrial", "<b>Altlantic</b> server is experiencing flooding issues", "Subject line!");
 
     private static final String CLUSTER_MANAGER_DEFAULT_EVENT_URL = "http://localhost/openshift/details/s/2XqNHRdLNEAzshh7MkkOql6fx6I?from=notifications&integration=teams";
+    public static final String EXPECTED_NOTIFICATION_TEXT_MESSAGE = "\"text\": \"1 event triggered from Cluster Manager - OpenShift. [Open Cluster Manager](" + CLUSTER_MANAGER_DEFAULT_EVENT_URL + ")\"";
 
     @Inject
     TemplateService templateService;
 
     @Inject
     ObjectMapper objectMapper;
 
-    @Test
-    void testRenderedOcmTemplates() {
-        String result = renderTemplate(null, MESSAGE);
-        checkResult(null, result);
+    @ValueSource(strings = { CLUSTER_MANAGER_CLUSTER_UPDATE, CLUSTER_MANAGER_CLUSTER_LIFECYCLE, CLUSTER_MANAGER_CLUSTER_CONFIGURATION,
```

Comment:
I changed a bit this test, because it's important to test every known event types rendered template outputs.